src/calibre/utils/chm/extra.c

```diff
@@ -42,6 +51,13 @@
 #                       define MODEXPORT(a) __declspec(dllexport) a
 #                       define MODIMPORT(a) extern a
 #               endif
+#define uint64_t unsigned long long
+#define uint32_t unsigned int
+#define uint16_t unsigned short
+#define uint8_t  unsigned char
+#define size_t   int
```

Comment:
this looks wrong. as of VS 2015 (which calibre uses on windows, all these types should be defined anyway, so no need to redefine them here.

Reply:
It is automagically generated by swig, probably under the assumption that people use very old compilers.

Reply:
Perhaps https://github.com/dottedmag/pychm/pull/5 is meant to work on very old compilers.

Reply:
yeah but even then size_t is an unsigned type while int is signed. That cant be right.